doc/topics/back-testing.md

```diff
@@ -1,9 +1,13 @@
 ## Back-testing
 
-*Before you start back-testing, you need to setup [MySQL](../../README.md#configure-mysql-database) or [SQLite3
+Currently bbgo supports two ways to run backtests:
+
+  1: Through csv data source (supported right now are binance, bybit and OkEx)
+
+  2: Alternatively run backtests through [MySQL](../../README.md#configure-mysql-database) or [SQLite3
```

Comment:
Would suggest to put mysql as the default backtest engine.

Let's put the csv data source as the second one